WordPress/JetpackStatsWidgets/LockScreenWidgets/Views/LockScreenMultiStatView.swift

```diff
@@ -31,9 +31,11 @@ struct LockScreenMultiStatView: View {
                     Spacer()
                 }
             }
+            .removableWidgetBackground()
             .accessibilityElement(children: .combine)
         } else {
             Text("Not implemented for widget family \(family.debugDescription)")
+                .removableWidgetBackground()
```

Comment:
Just curious about the Swift UI architecture, which is not my strong suit. Would it have been possible to group the conditional logic, e.g. in an `*Stack`, to only write the `removableWidgetBackground()` transformation once?

Reply:
Good question. I think it would be possible 👍